usdf/datasets.rst

```diff
@@ -2,49 +2,65 @@
 Common Dataset Organization and Policy
 ######################################
 
-This document covers the specific :ref:`format <format_usdf>` and :ref:`policies <policy_usdf>` governing the shared datasets in ``/datasets``, space available on the :doc:`login nodes </usdf/lsst-login>` and on all of the compute nodes of the :doc:`Batch Systems </usdf/batch>`.
+This document covers the specific :ref:`format <format_usdf>` and :ref:`policies <policy_usdf>` governing the shared datasets at the USDF, including space available on the :doc:`login nodes </usdf/lsst-login>` and on all of the compute nodes of the :doc:`Batch Systems </usdf/batch>`.
+
+Datasets covered by this policy include raws, calibration files, and refcats. "Refcats" refers to reference catalogs that are used for calibration (astrometric or photometric). Other types of catalogs may be used as references (e.g. DC2 truth tables) but will be referred to as external catalogs.
+
+File Paths
+==========
+
+.. _file_paths_usdf:
+
+The following file paths contain shared datasets:
+
+- ``/sdf/group/rubin/datasets`` (henceforth ``/datasets`` for short) is a symlink to ``/sdf/data/rubin/shared/ncsa-datasets``, containing datasets previously stored at NCSA under the now-defunct ``/datasets`` path (i.e. the ``/datasets`` symlink no longer exists at S3DF).
+- ``/sdf/group/rubin/shared`` (henceforth ``/shared`` for short) is a symlink to ``/sdf/data/rubin/shared`` and is the preferred path for new shared datasets, as well as for migrating older datasets.
+- ``/sdf/group/rubin/user`` is a symlink to ``/sdf/data/rubin/user`` and contains user home directories. Shared datasets may reside here temporarily for prototyping but should be moved to ``/shared`` once they start being used by multiple users.
+
+.. _policy_usdf:
+
+Policy
+======
+New shared datasets should be added to ``/shared``, following policies to be detailed in `DM-31704 <https://rubinobs.atlassian.net/browse/DM-31704>`_. Additions or modifications to any dataset that is to be used in regular (re-)processing must have a corresponding RFC. The RFC should contain information about:
```

Comment:
It's not exactly clear from this whether new shared datasets need an RFC.  We should say what the current status is, even if the policy is up in the air.  If we do require an RFC, then we should make clear that the request for space allocation below is after the RFC has been adopted.

Reply:
See the new line. I figure anything getting ingested must have an RFC, and anything being used for regular processing but not in any butler (I'm not sure what that would be, but maybe some camera/summit stuff?) ought to be too. Anything else I suppose doesn't need an RFC, although I'm not sure what the leaves. I guess common git repos like testdata_*?